### PR TITLE
Configure metrices

### LINKED ISSUE #425: Expose Metrics
# Rationale

We Should expose basic prometheus metrics for monitoring  and alerting.

With prometheus metrics we can get insight of applications like it's uptime, MATIC token balance, CPU/RAM usage.
# Implementation

Can follow  https://prometheus.io/docs/guides/go-application/

## Dev changes:

- [ ] Example code change

### PR DESCRIPTION
**Pull request by Shrikant1212**
# Description
 ## Expose metrics
We are exposing prometheus based metrics to monitor our staker, 

Fixes #425 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?
To expose a metrics we ca use razor cli todo so, `./razor setConfig --exposeMetrics 2112`

### CODE REVIEW

**Comment by Shrikant1212** [comment]
@ashish10677  @SkandaBhat  please help in reviewing this, willl update Radme.me soon.

**Review by Yashk767 — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -726,6 +726,32 @@ docker run -it  \
 
 Note : _All the commands have an additional --password flag that you can provide with the file path from which password must be picked._
 
+
+
+### Expose Metrics
+Can expose the prometheus based metrics for monitoring
```
Comment (SkandaBhat) [inline README.md:732]: Replace with "Expose Prometheus-based metrics for monitoring."

**Review by SkandaBhat — CHANGES_REQUESTED** [review]

**Review by SkandaBhat — APPROVED** [review]
lgtm

**Pull request merged**